src/browser-runner/platform-electron/sagas/server-command-handlers.js

```diff
@@ -86,6 +79,22 @@ function* openDevTools({ meta: client, payload: { winId, detach } }) {
   yield call([client, client.send], SharedActions.events.fromRunner.toServer.app.window.devtools.closed({ winId }));
 }
 
+function* registerKeyShortcuts({ meta: client, payload: { winId, shortcuts } }) {
```

Comment:
This is where we register shortcuts in electron-runner. These commands/actions originate in the browser frontend.